packages/beacon-node/src/eth1/provider/eth1Provider.ts

```diff
@@ -76,6 +78,8 @@ export class Eth1Provider implements IEth1Provider {
       // Don't fallback with is truncated error. Throw early and let the retry on this class handle it
       shouldNotFallback: isJsonRpcTruncatedError,
       jwtSecret: opts.jwtSecretHex ? fromHex(opts.jwtSecretHex) : undefined,
+      jwtId: opts.jwtId ?? undefined,
+      jwtVersion: opts.jwtVersion ?? undefined,
```

Comment:
Why is this required? Omitting `?? undefined` should have the same result

```suggestion
      jwtId: opts.jwtId,
      jwtVersion: opts.jwtVersion,
```